### PR TITLE
Created new component EventCard for matrix view on landing page, added style

### PR DESCRIPTION
**Pull request by SvitlanaYaroslavicka1**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/event/EventMatrix/EventCardV2.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/main.scss";
+
+.event-card {
+    &__body{
```
Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.scss]: Please adjust EsLint rules to indicate, that there should be one space between selector name and curly bracket

File: src/components/event/EventMatrix/EventMatrix.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.event-matrix{
+    &__body {
+    display: flex;
+    justify-content: space-evenly;
+    max-width: 1168px;
+    max-height: 531px;
+    margin: 0px auto;
+
```
Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventMatrix.scss]: Remove this empty line before curly bracket

File: src/components/event/EventMatrix/EventCardV2.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    &__inner-container {
+    display: inline-block;
+    position: absolute;
+    left: 20px;
+    top: 142px;
+    height: 84px;
+    width: 340px;
+    border-radius: 20px;
+    background-color: rgba(0,0,0,0.5);
+
```
Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.scss]: Remove this empty line before curly bracket

Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.scss]: Remove this empty line before curly bracket

File: src/components/event/EventMatrix/EventCardV2.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./EventCardV2.scss";
+
+const EventCard = ({ title, photo }) => {
+  return (
+    <div className="event-card">
```
Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.js:7]: Reduce count of these div tags

Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.js:7]: Reduce count of these div tags

Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.js:7]: Reduce count of these div tags

Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.js:7]: Reduce count of these div tags

File: src/components/event/EventMatrix/EventCardV2.scss (hunk 1 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.scss]: ок

File: src/components/event/EventMatrix/EventCardV2.scss (hunk 3 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.scss]: ок

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.scss]: ок

File: src/components/event/EventMatrix/EventMatrix.scss (hunk 2 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventMatrix.scss]: ок

File: src/components/event/EventMatrix/EventCardV2.scss (hunk 3 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.scss:25]: ок

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.scss:25]: ок

File: src/components/event/EventMatrix/EventCardV2.js (hunk 4 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js:7]: ок

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js:7]: ок

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js:7]: ок

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js:7]: ок

File: src/components/event/EventMatrix/EventCardV2.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      <img className="event-card__poster" src={photo} alt="Poster"></img>
+      <div className="event-card__inner-container">
+        <div className="event-card__text-container">
```
Comment (VadymKharyton) [inline src/components/event/EventMatrix/EventCardV2.js]: Inner container and text containers divs are redundant

Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js]: I agree about "event-card__text-container" but  "event-card__inner-container" needed for transparent background area

Comment (VadymKharyton) [reply src/components/event/EventMatrix/EventCardV2.js]: Right, I actually meant to make only div and assign all styles to this div

**Review by VadymKharyton — APPROVED** [review]

File: src/components/event/EventMatrix/EventCardV2.js (hunk 5 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/event/EventMatrix/EventCardV2.js]: ок

**Pull request merged**